apps/site/pages/en/learn/asynchronous-work/understanding-processnexttick.md

```diff
@@ -51,4 +51,13 @@ Running before the timeout => number 3
 The timeout running last => number 4
 ```
 
-The exact output may differ from run to run.
+The output aforementioned holds true in ES Modules, e.g. mjs files, but keep in mind in CommonJS case, the output may be different:
+
+```bash
```

Comment:
This is not valid bash syntax at all (would be worth fixing the above block as well)
````suggestion
```text
````

Reply:
in addition the example output above can be updated

Reply:
@AugustinMauroy, Syntax revised.